smarta/smarta/StationLinesTableViewController.swift

```diff
@@ -108,6 +111,20 @@ class StationLinesTableViewController: UITableViewController {
         }
     }
     
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        if searchText.count == 0 {
+            displayedStations = stations
+        } else {
+            displayedStations = stations.filter({ (station) -> Bool in
+                let tmp: NSString = station.station.name as NSString
+                let range = tmp.range(of: searchText, options: NSString.CompareOptions.caseInsensitive)
+                return range.location != NSNotFound
+            })
+        }
+
+        self.tableView.reloadData()
```

Comment:
This is happening on the main thread, so you wrap it like such.
```suggestion
DispatchQueue.main.async {
        self.tableView.reloadData()
        }
```

Reply:
ahh roger -- joe brought up the main queue vs. other queues.  Will apply this change!